piet/src/image.rs

```diff
@@ -22,6 +22,12 @@ use crate::kurbo::Size;
 use crate::util::unpremul;
 use crate::{Color, ImageFormat, RenderContext};
 
+/// Represents an image type that knows its size.
```

Comment:
```suggestion
/// A trait for a backend's bitmap image type.
```

Don't love my version either, but in general I'd try and make the docs slightly higher level.